MetadataExtractor/Formats/QuickTime/QuickTimeMetadataReader.cs

```diff
@@ -120,6 +120,23 @@ void UuidHandler(AtomCallbackArgs a)
                 }
             }
 
+            void UserDataHandler(AtomCallbackArgs a)
+            {
+                switch (a.TypeString)
+                {
+                    case "?xyz":
+                        var directory = new QuickTimeMetadataHeaderDirectory();
+                        var stringSize = a.Reader.GetUInt16();
+                        var languageCode = a.Reader.GetUInt16();
+                        var stringValue = a.Reader.GetBytes(stringSize);
+                        var s = Encoding.UTF8.GetString(stringValue);
+
+                        directory.Set(QuickTimeMetadataHeaderDirectory.TagGpsLocation, s);
```

Comment:
Can this string be parsed into a `GeoLocation`? It seems the value can have three components. Is the third altitude?

Even if not prased, the string should be stored as a `StringValue` so that consumers may reinterpret the raw bytes using a different encoding. I know that seems unlikely here, but it's our standard approach for storing parsed string values and gives people an escape hatch when they have non-standards-compliant data.

Reply:
In our sample files, the string is ISO 6709 format, such as "+37.7007-121.9416/".  I believe that's what TagGpsLocation already uses?